torch/fx/experimental/symbolic_shapes.py

```diff
@@ -772,6 +773,14 @@ def has_free_symbols(val: IterateExprs) -> bool:
     return not all(e.is_number for e in _iterate_exprs(val))
 
 
+def has_free_unbacked_symbols(x: IterateExprs) -> bool:
+    """Faster version of bool(free_unbacked_symbols(val))"""
+    for s in free_symbols(x):
+        if symbol_is_type(s, (SymT.UNBACKED_INT, SymT.UNBACKED_FLOAT)):
+            return True
```

Comment:
Following might be faster. Not sure.
```suggestion
    from sympy.core.traversal import iterargs
    for s in _iterate_exprs(x):
       for arg in iterargs(s):
          if isinstance(arg, sympy.Symbol) and symbol_is_type(arg, (SymT.UNBACKED_INT, SymT.UNBACKED_FLOAT)):
            return True
```

Reply:
i will try it out